webapi/Options/PlannerOptions.cs

```diff
@@ -11,20 +11,22 @@ namespace CopilotChat.WebApi.Options;
 public class PlannerOptions
 {
     /// <summary>
-    /// Whether to allow missing functions in plan on creation. If allowed, proposed plan will be sanitized of no-op functions.
-    /// Functions are considered missing if they're not available in the planner's kernel's context.
+    /// Options to handle planner errors.
     /// </summary>
-    public class MissingFunctionErrorOptions
+    public class PlannerErrorOptions
     {
         /// <summary>
-        /// Flag to indicate if skips are allowed on MissingFunction error
-        /// If set to true, the plan will be created with missing functions as no-op steps that are filtered from the final proposed plan.
-        /// If this is set to false, the plan creation will fail if any functions are missing.
+        /// Whether to allow retries on planner errors.
         /// </summary>
         public bool AllowRetries { get; set; } = true;
 
         /// <summary>
-        /// Max retries allowed on MissingFunctionsError.
+        /// Whether to allow missing functions.
```

Comment:
You probably want to keep the description of what this boolean does.  It's not obvious for people who don't know about it.